packages/store/tsconfig.json

```
@@ -2,6 +2,10 @@
   "extends": "@northern.tech/typescript-config/react-app.json",
   "exclude": ["dist", "build", "node_modules", "**/*.test.js"],
   "compilerOptions": {
+  
+      "skipLibCheck": true,
+      "noEmitOnError": false,
+ 
     "rootDir": ".",
```

```suggestion
  "compilerOptions": {
    "noEmitOnError": false,
    "rootDir": ".",
```
`skipLibCheck` is already defined via `react-app` => `base` + the formatting should be aligned

---

leftovers from the previous tries... I will remove them